build.sbt

```diff
@@ -1,34 +1,72 @@
+import sbt.Project.projectToRef
+
+lazy val clients = Seq(client)
```

Comment:
I think val clients should be moved closer to its usage (i.e. before server val)